documentation/dev/spec/protocol/inventory.md

```diff
@@ -50,7 +50,7 @@ title: Inventory protocol
   LOCAL_GROUPS+, LOCAL_USERS+, LOGICAL_VOLUMES+, MEMORIES+, MODEMS+, FIRMWARES+,
   MONITORS+, NETWORKS+, PHYSICAL_VOLUMES+, PORTS+, PRINTERS+, PROCESSES+, SLOTS+,
   SOFTWARES+, USERS+, STORAGES+, SOUNDS+, VIRTUALMACHINES+,
-  VOLUME_GROUPS+, REMOTE_MGMT+, SIMCARDS+, SENSORS+)>
+  VOLUME_GROUPS+, REMOTE_MGMT+, SIMCARDS+, SENSORS+, CAMERAS+)>
```

Comment:
Hi @Naylin15 
sorry, yesterday I found there was missing a lot of entry here and I pushed a commit to fix that. Can you rebase and add your `CAMERAS+` again at the end of the list ?